meta/internal/meta.pb.go

```diff
@@ -1378,15 +1414,11 @@ func (m *JoinRequest) GetAddr() string {
 }
 
 type JoinResponse struct {
-	Header *ResponseHeader `protobuf:"bytes,1,req" json:"Header,omitempty"`
-	// Indicates that this node should take part in the raft cluster.
```

Comment:
@jwilder @benbjohnson any idea how comments got in this file to begin with?  Should the comments exist in the meta file instead that generates this?